src/riak_pb_ts_codec.erl

```diff
@@ -83,21 +75,21 @@ decode_columns(Columns) ->
 -spec encode_rows(list(list({binary(), ldbvalue()}))) -> [#tsrow{}].
 %% @ignore copied from riakc_ts_put_operator; inverse of make_data
 encode_rows(Measurements) ->
-    rows_for(Measurements, []).
+    lists:map(fun encode_row/1, Measurements).
```

Comment:
I am running this by @jonmeredith. In the Riak code base it appears that list comprehension is the preferred style.

```
lbakken@brahms ~/Projects/basho/riak (develop=)
$ find . -type f -name '*.erl' -exec fgrep lists:map {} + | wc -l
164
lbakken@brahms ~/Projects/basho/riak (develop=)
$ find . -type f -name '*.erl' -exec egrep '\[.*\|\|.*<-.*\]' {} + | wc -l
```